src/components/Alert/common.ts

```diff
@@ -17,7 +17,7 @@ export interface StatefulAlertProps {
 }
 
 export const statefulPropTypes = {
-    variant: PropTypes.oneOf(['error', 'info', 'message', 'success']).isRequired,
+    variant: PropTypes.oneOf(['error', 'danger', 'info', 'message', 'success']).isRequired,
```

Comment:
I think we should go ahead and remove the `error` variant as long as we mention it in the commit message for the release notes.